server.py

```diff
@@ -31,16 +31,28 @@ def reservations_get():
         resp = Response(js, status=200, mimetype='application/json')
         return resp
 
+# Validate JSON data
+def is_json(myjson):
+  try:
+    json.loads(myjson.data.decode('utf-8'))
+  except ValueError:
+    return False
+  return True
+
 @app.route('/reservations', methods = ['POST'])
 def reservations_post():
     payload = request.get_json()
-    print(payload['hour'])
-    if not request.is_json:
+    print(payload)
+    if is_json(request) is False:
+        print(payload)
```

Comment:
You can remove these `print` lines, I'm guessing you added that to debug

Reply:
Oops. Thanks.